MitigationApi.yml

```diff
@@ -598,6 +598,18 @@ components:
       - DENIED

```

Comment:
This won't let me comment on an unchanged line, but line 8: need to bump the version!